packages/ui-shared/src/translations/index.tsx

```diff
@@ -102,6 +112,12 @@ const lang = (id: string | MultiLangText, values?: { [variable: string]: any }):
     return id[locale] || id[defaultLocale] || ''
   }
 
+  // pattern is isolangs.{code}
+  if (id.startsWith(ISO_LANG_PREFIX)) {
+    const code = id.substring(ISO_LANG_PREFIX.length, ISO_LANG_PREFIX.length + 2)
```

Comment:
Are you sure the code will always be two chars? Otherwise, you could simply remove the prefix from the string.

Reply:
yes 100% sure. cannot only remove prefix from string because we sometimes add .name after.